blocks/nautilus/ff_container/ff_container-dashboard-section/ff_container-dashboard-section.less

```diff
@@ -20,8 +17,9 @@
         font-size: 21rem/@ff_base;
     }
     &__main {
-        #ff_mix-sections > .main();
-        background-color: white;
+       #ff_mix-sections > .main();
+       padding: 0 @ff_size_control_horizontal;
```

Comment:
@boonier so this *does* include L/R padding, or not? Can I merge this in or now now? Prulease?